packages/discussions/src/utils/platform.ts

```diff
@@ -1,4 +1,16 @@
 import { GroupMembership, IGroup, IUser } from "@esri/arcgis-rest-portal";
+import {
+  AclCategory,
+  IChannelAclPermission,
+  IChannelAclPermissionDefinition,
+} from "../types";
+
+type PermissionsByAclCategoryMap = {
+  [key in AclCategory]?: IChannelAclPermission[];
+};
+type PermissionDefinitionsByAclCategoryMap = {
+  [key in AclCategory]?: IChannelAclPermissionDefinition[];
+};
```

Comment:
We shouldn't store anything related to ChannelAcl in the `platform.ts` file.  "platform" refers to sharing model used across ArcGIS and channelAcl isn't a part of that.